pkg/splunk/splunk.go

```diff
@@ -96,29 +92,12 @@ func RetrieveSearchFromAlert(sid string) (Alert, error) {
 	}
 
 	// Process the response
-	err = helpers.DecodeJSONResponseBody(resp, &searchResults)
+	err = helpers.DecodeJSONResponseBody(resp, &alert.SearchResults)
 	if err != nil {
 		return alert, err
 	}
 
-	alert.SearchResults = searchResults
-
-	log.Printf("Received alert from Splunk: %s", alert.SearchID)
-	for _, result := range alert.Details() {
-		log.Println(result)
-	}
-
-	// TODO: Can we make the un-marshalling of the XML response agnostic to any specific service?  Interfaces?
-	//var search Results
-
-	os.Exit(1)
-
+	log.Printf("Received alert from Splunk: %s, %v", alert.SearchID, alert.Details())
```

Comment:
```suggestion
	log.Printf("Retrieved alert details from Splunk: %s, %v", alert.SearchID, alert.Details())
```

Just to differentiate it from the receipt of the webhook, which also has an almost identical log message.